packages/firefuel/test/src/order_by_test.dart

```diff
@@ -5,6 +5,14 @@ import 'package:firefuel/firefuel.dart';
 void main() {
   const String testField = 'testField';
   group('$OrderBy', () {
+    group('#docId', () {
+      test('should set byId to true ', () {
+        final instance = OrderBy.docId(OrderDirection.aToZ);
+
+        expect(instance.byId, isTrue);
+      });
```

Comment:
This test will be dropped once you make the `byId` property private. You'll want to make sure that the documents you get back are ordered by documentId ascending, and then descending in a separate test.